website/static/js/project-organizer.js

```diff
@@ -0,0 +1,488 @@
+/**
```

Comment:
I see this is just a renaming of the old file.

Feel free to ignore my comments